sdk/python/feast/offline_server.py

```diff
@@ -342,6 +356,11 @@ def get_historical_features(self, command: dict, key: str):
             name_aliases=name_aliases,
             project=project,
         )
+        # Permission check for each feature view
+        for feature_view in feature_views:
+            assert_permissions(
```

Comment:
you can avoid the `for` loop as `assert_permissions` accepts a list of resources

Reply:
used the `permitted_resources` take a look.

Reply:
(it was an outdated comment sent before I introduced the filtering function 😉 )